Sources/AnimalKingdomAPI/animalkingdom-graphql/AnimalSchema.graphqls

```diff
@@ -41,15 +41,15 @@ input PetSearchFilters {
   measurements: MeasurementsInput
 }
 
-interface Animal {
+interface Animal @typePolicy(keyFields: "id") {
   id: ID!
   species: String!
   height: Height!
   predators: [Animal!]!
   skinCovering: SkinCovering
 }
 
-interface Pet {
+interface Pet @typePolicy(keyFields: "id") {
```

Comment:
I added the directive to one of the schemas, but I didn't regenerate the example sources since that would add another 100 files to the Pull Request